plugin/src/create-schema-customization.ts

```diff
@@ -188,50 +235,18 @@ export function createSchemaCustomization(
     );
   }
 
-  if (pluginOptions.downloadImages) {
+  typeDefs.push(
+    ...[
+      "ShopifyProductFeaturedImage",
+      "ShopifyProductFeaturedMediaPreviewImage",
+      "ShopifyProductVariantImage",
+    ].map((typeName) => defineImageNode(name(typeName), schema, pluginOptions))
```

Comment:
I like this 👍 much cleaner. One thing that might be nice to change is the function name of `defineImageNode` it'd be a bit clearer as `buildImageNodeObjectType` or `buildImageNodeType`. I think it's technically not defined until we pass it back to schema customization.